internal/cli/atlas/dbusers/create.go

```diff
@@ -166,13 +177,12 @@ func CreateBuilder() *cobra.Command {
 			}
 			opts.roles = append(opts.roles, args...)
 
-			if err := opts.validate(); err != nil {
-				return err
-			}
-
-			return opts.Prompt()
+			return opts.validate()
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := opts.Prompt(); err != nil {
```

Comment:
this was an anoying bug where cobra validates flags after the `PreRun` so if you were missing the username we'd still ask for a password